.github/styles/Vocab/FluxNinja/accept.txt

```diff
@@ -50,6 +50,20 @@ Prometheus
 Pub/Sub
 [Aa]uto-scalers?\b
 [Rr]ollouts?\b
+LinkedIn
+Nutanix
+Netsil
+SLAs
+end-user
+rpm
```

Comment:
We can just remove this from the list here. Why do we need to add it?

Reply:
Do you mean why we're adding words here in the aperture repo `accept.text`? 

Hardik has enabled folder sync from aperture repo to aperture-tech-repo so we have to add those words here as well, otherwise accept.text file in aperture-tech-docs will get overwritten.

Reply:
Ai mean we can remove the word end user.

Reply:
Oh, i see it's redundant. It is already within package's styling guidelines.